### DISCUSSION TITLE
postUpgradeTasks requires allowedPostUpgradeTasks which is not documented

### DISCUSSION THREAD

**Discussion by jase88** [renovatebot/renovate#9790] [Request Help] [answered]
https://docs.renovatebot.com/configuration-options/#postupgradetasks says

> Each command must match at least one of the patterns defined in allowedPostUpgradeTasks in order to be executed.

I can't find any configuration for _allowedPostUpgradeTasks_. Or are the commands within the configuration of postUpgradeTasks are meant? But they are not really defining a pattern, right?

**Comment by rarkins** [comment, marked as answer]
`allowedPostUpgradeCommands` (note: not Tasks) is an admin-only option. As in, the bot administrator must allow them, and you cannot "self approve" using repository config. That doc has a typo in the name and needs updating. Docs: https://docs.renovatebot.com/self-hosted-configuration/#allowedpostupgradecommands

**Reply by rarkins** [reply]
I've pushed a commit to update the docs, it should flow through to the website later today

**Reply by jase88** [reply]
Thank you! now things are clearer for me 🙂
